### ISSUE TITLE
Running on Mac M1 (darwin-arm64 architecture)

### ISSUE THREAD

**Issue by avh4** [avh4/elm-format#722]
This error has been reported:

```
Running elm-format for the first time; downloading the actual binary
No binaries are available for your platform: darwin-arm64
```

TODO:
- [x] **help wanted**: does the [mac-x64 binary](https://github.com/avh4/elm-format/releases/download/0.8.4/elm-format-0.8.4-mac-x64.tgz) run on these platforms?
- [x] if so, then the npm installer should be updated to map `darwin-arm64` to the existing binary

Future work:
- make an actual darwin-arm64 release -> https://github.com/avh4/elm-format/issues/723

**Comment by lydell** [comment]
It seems to work on M1:

<img width="426" alt="image" src="https://user-images.githubusercontent.com/2142817/106880056-32707080-66dc-11eb-90a0-5cb83b43921c.png">

**Comment by avh4** [comment]
I just published a new release candidate -- can someone check if `npm install elm-format@0.8.5-rc1` gets working elm-format on an M1 Mac?  (It should be expected that Rosetta will need to be installed for it to work.)

**Comment by theothornhill** [comment]
With `npm i -g elm-format@0.8.5-rc1` formatting works for me under M1.

- nvm compiled through normal terminal
- `npm i -g elm elm-test @elm-tooling/elm-language-server`
- emacs lsp-mode picks it up
- works from native terminal
- emacs runs arm


<img width="308" alt="image" src="https://user-images.githubusercontent.com/26711805/107116336-ae9cbc80-6872-11eb-80dc-82ff9f47aaae.png">

<img width="827" alt="image" src="https://user-images.githubusercontent.com/26711805/107116347-c4aa7d00-6872-11eb-9f7e-6da114000623.png">

**Comment by avh4** [comment]
elm-format@0.8.5 should work on the new Macs.  For native arm64 binaries, see #723 (and the upstream blockers)

**Comment by segmentationfaulter** [comment]
it works with npm installer, but not working for me with yarn installer.

**Comment by avh4** [comment]
I'm not familiar with yarn... any idea what it does differently?  (And can you post the error output of trying to install with yarn?)

**Comment by segmentationfaulter** [comment]
@avh4 Attaching the error output with `yarn`

<img width="1125" alt="Screenshot 2021-12-22 at 9 03 17 AM" src="https://user-images.githubusercontent.com/6733537/147033776-67681a74-7a20-41f3-8b57-1434b7db00ad.png">

**Comment by avh4** [comment]
Which version of elm-format did it download?  That error "No binaries are available for you platform: darwin-arm64" should be fixed in 0.8.5.

**Comment by segmentationfaulter** [comment]
Sorry for the false alarm, I had a outdated dependency which had an outdated dependency of elm-format, updating my dependencies made it work.
